enterprise_access/apps/content_assignments/constants.py

```diff
@@ -34,12 +34,14 @@ class AssignmentActions:
     NOTIFIED = 'notified'
     REMINDED = 'reminded'
     CANCELLED_NOTIFICATION = 'cancelled'
+    DEADLINE_PASSED_NOTIFICATION = 'deadline_passed'
```

Comment:
Good idea.  I might generalize this name/string a little bit to be something like "automatic_cancellation_notification"